src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -214,8 +214,116 @@ As AWS is a paying service, when the Scheduler is stopped normally (without remo
 
 WARNING: If ProActive server is forcibly killed, the created AWS instances will not be terminated. And, when ProActive server is restarted, the infrastructure will be re-configured as per the previous settings. If the instances were deleted at the AWS side, they will be re-created and re-configured.
 
+==== AWS Autoscaling Infrastructure
 
+Similarly to the EC2 Infrastructure, the *AWS Autoscaling Infrastructure* operates AWS EC2 service to provide computing node to the Resource Manager.
+However, it implements a different instance management from it to lower the delay of node acquisition and node removal process and ease inter-node collaboration in the same cluster.
 
```

Comment:
it implements ... and ease ... the same cluster **, thanks to the following changes:**

Reply:
Done